apps/user-profile/src/app/pages/settings-page/settings-mailing-lists/settings-mailing-lists.component.ts

```diff
@@ -27,25 +31,93 @@ export class SettingsMailingListsComponent implements OnInit {
   index: number;
   filteredVos: Vo[] = [];
   loading: boolean;
+  selectedVo: string = null;
+  selectedResource: string = null;
+  changeOptOut: string;
 
   constructor(
     private store: StoreService,
     private usersManagerService: UsersManagerService,
     private membersService: MembersManagerService,
+    private vosManagerService: VosManagerService,
     private resourcesManagerService: ResourcesManagerService,
-    private attributesManagerService: AttributesManagerService
+    private attributesManagerService: AttributesManagerService,
+    private route: ActivatedRoute,
+    private router: Router,
+    private location: Location,
```

Comment:
this service is not used anywhere, please remove it from constructor and don't forget to remove it from imports